Tasks/DockerV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -20,6 +20,8 @@
   "loc.input.help.tags": "A list of tags in separate lines. These tags are used in build, push and buildAndPush commands. Ex:<br><br>beta1.1<br>latest",
   "loc.input.label.arguments": "Arguments",
   "loc.input.help.arguments": "Docker command options. Ex:<br> For build command,<br>--build-arg HTTP_PROXY=http://10.20.30.2:1234 --quiet",
+  "loc.input.label.hidePII": "Hide identifiable information from image metadata",
+  "loc.input.help.hidePII": "Prevent publishing identifiable information to image metadata store",
```

Comment:
By default pipeline data like source branch name, build id are added which helps with trace-ability. For example you can inspect an image to find out which pipeline built the image. You can opt out of this default behavior by using this input.